src/main/java/io/cdap/plugin/successfactors/common/exception/RetryableException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.successfactors.common.exception;
+
+/**
+ * Custom Exception Class for handling retrying API calls
+ */
+public class RetryableException extends RuntimeException {
```

Comment:
why are you creating this class? Doesn't it conflict with io.cdap.cdap.api.retry.RetryableException (used in SuccessFactorsTransporter)?

Reply:
Yes, removed the `RetryableException` class. 56a52fdfdf35f6413fcb96fa9b6f3e475dc6124d